ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -312,10 +315,24 @@ object Config {
           success
       })
 
+      opt[Unit]("early-access")
```

Comment:
why do we want this to be inconsistent with sandbox? That just seems annoying. And it really is unsafe.

Reply:
cc @mziolekda